src/main/java/uk/gov/justice/digital/hmpps/csr/api/model/DetailNotification.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.csr.api.model
+
+import java.time.LocalDate
+import java.time.LocalDateTime
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Id
+
+data class DetailNotification(
```

Comment:
From reading the naming, I cannot figure out what the difference between a Detail and a DetailNotification is